Plugins/Disk/Retrieve-MFT.ps1

```diff
@@ -39,10 +39,10 @@ param (
 
 process{
 
-   # Verify that 7za executables are located in $global:PowerResponse.Config.Path.Bin
+   # Verify that 7za executables are located in (Get-PRPath -Bin)
 
-    $7za32 = ("{0}\7za_x86.exe" -f $global:PowerResponse.Config.Path.Bin)
-    $7za64 = ("{0}\7za_x64.exe" -f $global:PowerResponse.Config.Path.Bin)
+    $7za32 = ("{0}\7za_x86.exe" -f (Get-PRPath -Bin))
+    $7za64 = ("{0}\7za_x64.exe" -f (Get-PRPath -Bin))
```

Comment:
The function does not appear to be working as intended. I am getting errors for finding things in BIN.

Reply:
I re-added the global scoping to `$PowerResponse` and this works again :( sad I can't down scope that easily, but I feel better about plugin developers using this function rather than an obscure global variable.